src/Stryker.Core/Stryker.Core/Mutators/IntegerNullificationMutator.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+
+namespace Stryker.Core.Mutators;
+
+public class IntegerNullificationMutator : MutatorBase<LiteralExpressionSyntax>
```

Comment:
Why having two separate mutators ?

Reply:
I figured they can possibly be conflicting, and so I thought it'd be a good idea to add them separately. 
When mutating `var x = 10` for instance, would this be `var x = 0`? `var x = -10`?
Maybe I'm all wrong about this and it doesn't work this way at all: if this is not an issue it could be a single mutator.